cukas.js

```diff
@@ -4,6 +4,7 @@ class CukasGame {
     trump = null; // Card
     attack = []; // Attack cards
     defence = []; // Defence cards
+    hands = [];
```

Comment:
Būtu zolīdi atstāt arī komentāru: Array of CardSet objects, representing player hands.

Reply:
jau atstaaju